src/forms/create_ticket.ts

```diff
@@ -34,7 +34,7 @@ export async function newCreateTicketForm(call: AppCallRequest): Promise<AppForm
         botAccessToken: context.bot_access_token,
         adminAccessToken: context.admin_access_token,
     };
-    const mmClient = newMMClient(mmOptions).asAdmin();
+    const mmClient = newMMClient(mmOptions).asActingUser();
```

Comment:
The token is already getting expanded in https://github.com/mattermost/mattermost-app-zendesk/blob/3ed148a73f308404cc52709d4d1cb842f3eba4ab/src/bindings/bindings.ts#L146-L146